spec/features/bibliography_resource_integration_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Bibliography resource integration test', type: :feature do
+  subject(:bibliograpy_resource) { BibliographyResource.new(url: 'article.bib', exhibit: exhibit) }
+
+  let(:exhibit) { FactoryGirl.create(:exhibit) }
+  let(:title_fields) do
+    %w(title_245a_search title_245_search title_sort title_display title_full_display)
+  end
+
+  it 'can write the document to solr' do
+    bibliograpy_resource.reindex
```

Comment:
this is a bit unclear why there's not an `expect` assertion here

Reply:
Gotcha I guess it is just so that an error doesn't raise.

Reply:
in that case, i think you can do `expect { bibliography_resource.reindex }.not_to raise_error`

Reply:
Of course, good catch. Added.